src/BotConnection.ts

```diff
@@ -75,7 +75,23 @@ export interface Receipt {
         total?: string,
         buttons?: Button[]
     }
-}    
+}  
+
+// only supported for 'skype' channel.
+export interface FlexCard {
+    contentType: "application/vnd.microsoft.card.flex",
+    content: {
+        title?: string,
+        subtitle?: string,
+        text?: string,
+        images?: { 
+            url: string,
+            tap?: Button 
+        }[],
```

Comment:
Truly a small nit, but to keep consistency with the other attachment definitions, please put this whole image definition on one line.